src/FluxCalculator.jl

```diff
@@ -304,19 +337,20 @@ function get_surface_fluxes!(inputs, surface_params::SF.Parameters.SurfaceFluxes
     F_turb_moisture = SF.evaporation.(surface_params, inputs, outputs.Ch)
 
     # Zero out fluxes where the area fraction is zero
-    @. F_turb_ρτxz = ifelse(area_fraction == 0, zero(F_turb_ρτxz), F_turb_ρτxz)
-    @. F_turb_ρτyz = ifelse(area_fraction == 0, zero(F_turb_ρτyz), F_turb_ρτyz)
-    @. F_shf = ifelse(area_fraction == 0, zero(F_shf), F_shf)
-    @. F_lhf = ifelse(area_fraction == 0, zero(F_lhf), F_lhf)
-    @. F_turb_moisture = ifelse(area_fraction == 0, zero(F_turb_moisture), F_turb_moisture)
+    @. F_turb_ρτxz = ifelse(area_fraction ≈ 0, zero(F_turb_ρτxz), F_turb_ρτxz)
+    @. F_turb_ρτyz = ifelse(area_fraction ≈ 0, zero(F_turb_ρτyz), F_turb_ρτyz)
+    @. F_shf = ifelse(area_fraction ≈ 0, zero(F_shf), F_shf)
+    @. F_lhf = ifelse(area_fraction ≈ 0, zero(F_lhf), F_lhf)
+    @. F_turb_moisture = ifelse(area_fraction ≈ 0, zero(F_turb_moisture), F_turb_moisture)
 
-    return (;
-        F_turb_ρτxz = F_turb_ρτxz,
-        F_turb_ρτyz = F_turb_ρτyz,
-        F_shf = F_shf,
-        F_lhf = F_lhf,
-        F_turb_moisture = F_turb_moisture,
-    )
+    L_MO = outputs.L_MO
+    # We need to be careful with L_MO because `Inf` is valid output
+    L_MO = ifelse.(isinf.(outputs.L_MO), zero(outputs.L_MO), outputs.L_MO)
```

Comment:
Why do we need this?

Reply:
We probably don't need this, since we are already filtering for NaNs in a different part of the code

Reply:
I'll try removing it

Reply:
Is this setting Inf to zero for L_MO? It would be good if we can keep Inf in L_MO.

Reply:
Yes, I'll do that